spec/setup_acceptance_node.pp

```diff
@@ -1,4 +1,4 @@
-if $facts['os']['name'] == 'CentOS' {
+if $facts['os']['name'] == ['CentOS', 'AlmaLinux'] {
```

Comment:
This doesn't work because `$facts['os']['name']` is never an array. What I suggested is
```suggestion
if $facts['os']['family'] == 'RedHat' and $facts['os']['name'] != 'Fedora' {
```

Reply:
Hmm I was trying to come up with something like this, but thanks!